.azure/pipelines/quarantined-tests.yml

```diff
@@ -1,13 +1,10 @@
-# We want to run quarantined tests on master as well as on PRs
-trigger:
-  batch: true
-  branches:
-    include:
-    - master
+# We only want to run quarantined tests on master
+pr: none
+trigger: none
 
 schedules:
-- cron: "0 */4 * * *"
-  displayName: Every 4 hours test run
+- cron: "0 23 * * *"
+  displayName: Run tests once a day at 11 PM
```

Comment:
Actually, this isn't 11 PM because the server time is offset about 5 hours from PDT. Use `0 18 * * *` instead if I'm reading the difference between `schedules` in helix-matrix.yml and when that pipeline actually runs correctly.

Reply:
Ok updated